ui/src/app/views/app/component.html

```diff
@@ -4,12 +4,20 @@
   <img class="portal-logo" src="assets/images/portal-logo.svg" alt="Researcher Portal"
     style="margin-left: 0.5rem; width: 13rem;">
 </div>
-<nav style="margin-top: 1rem;">
+<nav *ngIf="isSignedIn | async" style="margin-top: 1rem;">
   <a routerLink="/login" routerLinkActive="active">Switch User</a>
   <a routerLink="/repository" routerLinkActive="active">Select CDR</a>
 </nav>
 <!--
   Angular's builtin Router module will attach specific views here
   based on clicking routerLink elements (above) or router.navigate calls.
 -->
-<router-outlet></router-outlet>
+<div *ngIf="(user | async).isSignedIn">
```

Comment:
I think this would be slightly simpler with this pattern:

```
<div *ngIf="condition; else elseclausename">
  <p>Condition is true!</p>
</div>
<ng-template #elseclausename">
  <p>Condition is false!</p>
</ng-template>
```